components/outputs/image.qmd

```diff
@@ -65,7 +65,6 @@ listing:
         <p>Set `ui.output_image(Fill=True)` to allow the image to grow or shrink to fill its container as the app is resized.</p>
         
         <p>You can use an image as an input widget, collecting the locations of user clicks, double clicks, hovers, and brushes. To do this, follow the instructions provided for [plots](TO DO: ADD LIN KTO THE DETAILS SECTION OF THE PLOTS PAGE).</p>
-
 ---
```

Comment:
There's also a couple places that say "text" instead of "image":

Line 44: 

````
<li>Call `ui.output_image()` in the UI of your app to create a div in which to display the text. Where you call this function within the UI functions will determine where the text will appear within the layout of the app. Set the id argument of `ui.output_image()` to a unique value.</li>
````

should be:

````
 <li>Call `ui.output_image()` in the UI of your app to create a div in which to display the image. Where you call this function within the UI functions will determine where the image will appear within the layout of the app. Set the id argument of `ui.output_image()` to a unique value.</li>
````